third_party/libtock-drivers/src/usb_ctap_hid.rs

```diff
@@ -231,8 +231,9 @@ fn recv_with_timeout_detail(
     if matches!(status, Ok(SendOrRecvStatus::Timeout)) {
         #[cfg(feature = "verbose_usb")]
         writeln!(Console::new(), "Cancelling USB receive due to timeout").unwrap();
-        let result_code =
-            unsafe { syscalls::raw::command(DRIVER_NUMBER, command_nr::CANCEL, 0, 0) };
+        let result_code = unsafe {
+            syscalls::raw::command(DRIVER_NUMBER, command_nr::CANCEL, 0, 0)
+        };
```

Comment:
Do you have a different version of `cargo fmt`? We shouldn't have a diff here?

Also, this is the endpoint-free call to `CANCEL`, see the other comment.

Reply:
My mistake, I manually changed the wrong call to cancel.